src/library/librarycontrol.cpp

```diff
@@ -324,7 +325,23 @@ LibraryControl::LibraryControl(Library* pLibrary)
                 }
             });
 
-    /// Deprecated controls
+    // Show the track context menu for selected tracks, or hide it
+    // if it is the current active window
+    m_pShowTrackMenu = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "show_track_menu"));
```

Comment:
how about "toggle_track_menu" or "show_hide_track_menu"

Reply:
I removed that commit, it's now only in #4465